cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CookieJarFactory.java

```diff
@@ -34,21 +40,59 @@ private CookieJarFactory() {
         throw new AssertionError("Utility class");
     }
 
-    public static CookieJarBuilder inMemoryCookieJar() {
-        return new InMemoryCookieJarBuilder();
+    public static CookieJarSupplier inMemoryCookieJar() {
```

Comment:
Remove this if unneeded.

Reply:
I do need this for unit tests as I've kept the scope of the InMemoryCookieJarSupplier reduced.